File: Foundation/Scratch/Getting Started/c++/l001func.cpp
#include<iostream>
using namespace std;
void add(int a, int b)
{
    cout<<a+b<<endl;
}
void sub(int a, int b)
{
    cout<<b-a<<endl;
}
void multiply(int a, int b)
{
    cout<<a*b<<endl;
}
void divide(int a, int b)
{
    cout<<b/a<<endl;
}
int main()
{
    int a,b;
    cin>>a>>b;
    add(a,b);
    sub(a,b);
    multiply(a,b);
    divide(a,b);
    return 0;
}

File: Foundation/Scratch/Getting Started/java/l001func.java
import java.util.Scanner;
public class l001Func{
    public static Scanner scn = new Scanner(System.in);
    public static void add(int a, int b)
    {
      System.out.println(a+b);
    }
    public static void sub(int a, int b)
    {
      System.out.println(b-a);
    }
    public static void multiply(int a, int b)
    {
      System.out.println(a*b);
    }
    public static void divide(int a, int b)
    {
      System.out.println(b/a);
    }
    public static void main(String[] args)
    {
        int a = scn.nextInt();
        int b = scn.nextInt();
        add(a,b);
        sub(a,b);
        multiply(a,b);
        divide(a,b);
    }
}
